sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -209,7 +213,12 @@ export class EventHubClient {
    * @param credential - SharedKeyCredential object or your credential that implements the TokenCredential interface.
    * @param options -  A set of options to apply when configuring the client.
    */
-  constructor(host: string, eventHubPath: string, credential: TokenCredential, options?: EventHubClientOptions);
+  constructor(
+    host: string,
+    eventHubPath: string,
+    credential: TokenCredential,
+    options?: EventHubClientOptions
+  );
```

Comment:
Can you check if you are using the prettier extension or not? These formatting changes shouldnt be happening

Reply:
Removed